pkg/core/resources/model/resource.go

```diff
@@ -131,7 +131,9 @@ func Validate(resource Resource) error {
 
 func Deprecations(resource Resource) []string {
 	if v, ok := interface{}(resource).(interface{ Deprecations() []string }); ok {
-		return v.Deprecations()
+		deprecations := []string{fmt.Sprintf("%s type resource '%s' has the following fields deprecations: ",
```

Comment:
```suggestion
		deprecations := []string{fmt.Sprintf("%s type resource %q has the following fields deprecations: ",
```

`%q` adds quotes in a safe way.

Reply:
Hi @lahabana , I have updated the message.
Thanks a lot!